functorch/csrc/BatchRulesStopDecomposition.cpp

```diff
@@ -207,9 +200,7 @@ TORCH_LIBRARY_IMPL(aten, FT_BATCHED_KEY, m) {
   STOP_DECOMPOSE(diagflat);
   STOP_DECOMPOSE(diagonal.Dimname);
   STOP_DECOMPOSE(diff.out);
-  STOP_DECOMPOSE(divide.Scalar);
   STOP_DECOMPOSE(divide.Scalar_mode);
```

Comment:
divide.Scalar_mode and divide.Tensor_mode are also aliases, should we remove them as well ?

Reply:
Yup, go for it